R/helper_clean_model_data.R

```diff
@@ -142,6 +142,11 @@ clean_model_data <- function(data, datargs, estimator = "") {
   ret[["condition_pr"]] <- model.extract(mf, "condition_pr")
 
   ret[["fixed_effects"]] <- model.extract(mf, "fixed_effects")
+  # If there is NA in the blocks and only one block, returns vector not matrix
+  # so coerce to matrix
+  if (is.character(ret[["fixed_effects"]])) {
```

Comment:
to check if it's a matrix, usually you do `is.matrix` - since as.matrix is so cheap, may as well do it unconditionally though.